bin/crew

```diff
@@ -1147,6 +1149,9 @@ def resolve_dependencies
 
   @dependencies.each do |dep|
     depends = nil
+    unless File.exist?("#{CREW_PACKAGES_PATH}#{dep}.rb")
+      abort"Dependency #{dep} was not found.".lightred
```

Comment:
there's a missing whitespace right after `abort`. Does it even work without it? That'd be odd

Reply:
Fixed!